### ISSUE TITLE
Daylighting Sensor Controls

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#277]
### Description
Measure will place daylight sensors in zones that require lighting. The sensors will control the electric lights in the zones so that the use of daylight is maximized.

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	

|CEO  |P-004001.001 |Cost-Effective Pathways for NZE, NZE Ready, and GHG Neutral Buildings |Phylroy Lopez

**Comment by mlubun** [comment]
Adding the costing specification for daylighting controls in the thermal zones

**Comment by mlubun** [comment]
[BTAP-Daylighting Spec.docx](https://github.com/canmet-energy/btap_tasks/files/4773056/BTAP-Daylighting.Spec.docx)
